cypress.config.js

```diff
@@ -27,6 +28,17 @@ module.exports = defineConfig({
         }),
       );
 
+      on('task', {
+        deleteFile(filePath) {
+          if (fs.existsSync(filePath)) {
+            fs.unlinkSync(filePath);
+            return null;
+          } else {
+            throw new Error('File does not exist');
```

Comment:
If we try to delete a file and is not present, then the file is already "deleted", do nothing instead of failing.